react/typings/events.d.ts

```diff
@@ -161,31 +172,36 @@ export interface CartLoadedData extends EventData {
 export interface PromoViewData extends EventData {
   event: 'promoView'
   eventType: 'vtex:promoView'
+  eventName: 'vtex:promoView'
```

Comment:
This `eventType`/`eventName` repetition is essential because `eventName` is the property discriminator of the union types.